account/identity.go

```diff
@@ -0,0 +1,127 @@
+package account
+
+import (
+	"time"
+
+	"github.com/almighty/almighty-core/models"
+	"github.com/goadesign/goa"
+	"github.com/jinzhu/gorm"
+	uuid "github.com/satori/go.uuid"
+	"golang.org/x/net/context"
+)
+
+// Identity ddenDescribes a unique Person with the ALM
+type Identity struct {
+	models.Lifecycle
+	ID       uuid.UUID `sql:"type:uuid default uuid_generate_v4()" gorm:"primary_key"` // This is the ID PK field
+	Emails   []User    // has many Users
+	FullName string    // The fullname of the Identity
+	ImageURL string    // The image URL for this Identity
+}
+
+// TableName overrides the table name settings in Gorm to force a specific table name
+// in the database.
+func (m Identity) TableName() string {
+	return "identities"
+
+}
+
+// GormIdentityRepository is the implementation of the storage interface for
+// Identity.
+type GormIdentityRepository struct {
+	db *gorm.DB
+}
+
+// NewIdentityRepository creates a new storage type.
+func NewIdentityRepository(db *gorm.DB) IdentityRepository {
+	return &GormIdentityRepository{db: db}
+}
+
+// IdentityRepository represents the storage interface.
+type IdentityRepository interface {
+	Load(ctx context.Context, id uuid.UUID) (*Identity, error)
+	Create(ctx context.Context, identity *Identity) error
+	Save(ctx context.Context, identity *Identity) error
+	Delete(ctx context.Context, id uuid.UUID) error
+	Query(funcs ...func(*gorm.DB) *gorm.DB) ([]*Identity, error)
+}
+
+// TableName overrides the table name settings in Gorm to force a specific table name
+// in the database.
+func (m *GormIdentityRepository) TableName() string {
+	return "identities"
+
+}
+
+// CRUD Functions
+
+// Load returns a single Identity as a Database Model
+// This is more for use internally, and probably not what you want in  your controllers
+func (m *GormIdentityRepository) Load(ctx context.Context, id uuid.UUID) (*Identity, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "identity", "load"}, time.Now())
+
+	var native Identity
+	err := m.db.Table(m.TableName()).Where("id = ?", id).Find(&native).Error
+	if err == gorm.ErrRecordNotFound {
+		return nil, nil
```

Comment:
I have added a PR on [here](https://github.com/aslakknutsen/almighty-core/pull/3) for the same.